src/eth1spec/rlp.py

```diff
@@ -49,6 +51,18 @@ def encode(raw_data: RLP) -> Bytes:
         return encode_bytes(raw_data.encode())
     elif isinstance(raw_data, Sequence):
         return encode_sequence(cast(Sequence[RLP], raw_data))
+    elif isinstance(raw_data, Block):
+        return encode_block(raw_data)
+    elif isinstance(raw_data, Header):
+        return encode_header(raw_data)
+    elif isinstance(raw_data, Account):
+        return encode_account(raw_data)
+    elif isinstance(raw_data, Transaction):
+        return encode_transaction(raw_data)
+    elif isinstance(raw_data, Receipt):
+        return encode_receipt(raw_data)
+    elif isinstance(raw_data, Log):
+        return encode_log(raw_data)
```

Comment:
We've got two options here:
 - Either we need to add `Block`, `Header`, etc to the definition of the `RLP` type; or
 - We only handle tuples in `encode`, and create separate functions (ex. `block_to_tuple(...) -> Tuple[RLP]`) for each type.

Reply:
Oops, I forgot to add `Block` and `Header` to the definition of the `RLP` type. Solution 2 sounds good as well. I'm guessing that this would be rearchitected when we implement decode to the dataclasses.